.github/dependabot.yml

```diff
@@ -0,0 +1,27 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    # default location of `.github/workflows`
+    directory: "/"
+    open-pull-requests-limit: 10
+    schedule:
+      interval: "weekly"
+    target-branch: 'develop'
```

Comment:
`target-branch` is the branch which Dependabot would work against (default value is `main`)

Reply:
Sure 🙂